homeassistant/components/smartthings/sensor.py

```diff
@@ -526,3 +541,51 @@ def state(self):
             return three_axis[self._index]
         except (TypeError, IndexError):
             return None
+
+
+class SmartThingsPowerConsumptionSensor(SmartThingsEntity, SensorEntity):
+    """Define a SmartThings Sensor."""
+
+    def __init__(
+        self,
+        device: DeviceEntity,
+        report_name: str,
+    ) -> None:
+        """Init the class."""
+        super().__init__(device)
+        self.report_name = report_name
+
+    @property
+    def name(self) -> str:
+        """Return the name of the binary sensor."""
+        return f"{self._device.label} {self.report_name}"
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return f"{self._device.device_id}.{self.report_name}"
+
+    @property
+    def state(self):
+        """Return the state of the sensor."""
+        value = self._device.status.attributes[Attribute.power_consumption].value
+        if value.get(self.report_name) is None:
+            return None
+        elif self.report_name == "power":
+            return value[self.report_name]
+        else:
+            return value[self.report_name] / 1000
```

Comment:
```suggestion
        return value[self.report_name] / 1000
```
this will solve the pylint error ...  sorry, my bad in https://github.com/home-assistant/core/pull/53759#discussion_r680403974